### PR TITLE
atomic-openshift-install should only output relevant information

### PR DESCRIPTION
**Pull request by tbielawa**
With respect to https://github.com/openshift/openshift-ansible/issues/2445

### CODE REVIEW

**Comment by tbielawa** [comment]
aos-ci-test

File: utils/etc/ansible.cfg [hunk 1]
```diff
@@ -19,10 +19,15 @@ nocows = 1
...
 # callback_plugins - derive from pkg_resource.resource_filename
 # private_key_file - prompt if missing
 # remote_tmp - set if provided by user (cli)
 # ssh_args - set if provided by user (cli)
-# control_path
\ No newline at end of file
+# control_path
+
+stdout_callback = openshift_quick_installer
+callback_whitelist = openshift_quick_installer
```
Comment (detiber) [inline utils/etc/ansible.cfg]: Are these actually needed? I was under the assumption that callbacks only needed to be whitelisted when they are disabled by  default.

File: utils/etc/ansible.cfg [hunk 2]
```diff
@@ -19,10 +19,15 @@ nocows = 1
 
 retry_files_enabled = False
 
+deprecation_warnings=False
```
Comment (detiber) [inline utils/etc/ansible.cfg]: Is it not possible to filter these out in the callback plugin? Maybe it's not a big deal for the quick installer, but I know we are getting ready to hit a point where at least some of the deprecation warnings are going to be failures with a newer 2.2 build.

**Comment by tbielawa** [comment]
Rebased and squashed. Unit tests are not passing. Failures due to introduction of new `ansible-quiet.cfg` config file.

Still a work in progress. Handlers are not 'handled' yet and other acceptance criteria have not yet been met (such as making sure a log file of the complete run is still generated in regular ansible format)

**Comment by tbielawa** [comment]
aos-ci-test

**Comment by tbielawa** [comment]
@abutcher @smunilla @detiber  PTAL/Provide Feedbacks

**Comment by tbielawa** [comment]
@abutcher @smunilla @detiber can you guy try to review this PR today? Much thanks.

File: utils/src/ooinstall/cli_installer.py [hunk 3]
```diff
@@ -876,13 +870,15 @@ def cli(ctx, unattended, configuration, ansible_playbook_directory, ansible_conf
...
-        oo_cfg.settings['ansible_config'] = ctx.obj['ansible_config']
-    elif 'ansible_config' not in oo_cfg.settings and \
-         os.path.exists(DEFAULT_ANSIBLE_CONFIG):
+    # if ctx.obj['ansible_config']:
+    #     oo_cfg.settings['ansible_config'] = ctx.obj['ansible_config']
+    if os.path.exists(DEFAULT_ANSIBLE_CONFIG):
         # If we're installed by RPM this file should exist and we can use it as our default:
         oo_cfg.settings['ansible_config'] = DEFAULT_ANSIBLE_CONFIG
 
+    if os.path.exists(DEFAULT_ANSIBLE_CONFIG):
```
Comment (jiajliu) [inline utils/src/ooinstall/cli_installer.py]: Does "DEFAULT_ANSIBLE_CONFIG" here should be "QUIET_ANSIBLE_CONFIG"??

Comment (jiajliu) [inline utils/src/ooinstall/cli_installer.py]: Does "DEFAULT_ANSIBLE_CONFIG" should be "QUIET_ANSIBLE_CONFIG" here?

Comment (tbielawa) [inline utils/src/ooinstall/cli_installer.py]: Good catch!

**Comment by tbielawa** [comment]
aos-ci-test

**Comment by tbielawa** [comment]
Rebased onto latest master.

Fixed issue pointed out by @jiajliu 

Includes a conditional message at the end of the playbook run to let users know why plays may have been skipped. 

* https://gist.github.com/tbielawa/f904a5aab1966c2d7fac045732af5bc0#file-final-suggestion

> Installation Complete: Note: Play count is an estimate and some were skipped because your install does not require them

Also, removed the final 'press a key to continue' prompt.

@smunilla @abutcher @sdodson ready for review/merge after CI finishes.

**Pull request merged**